apps/backend/src/app/auctions/auctions.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { Controller, Get, Post, Body, Patch, Param, Delete } from '@nestjs/common';
+import { AuctionsService } from './auctions.service';
+import { CreateAuctionDto } from './dto/create-auction.dto';
+import { UpdateAuctionDto } from './dto/update-auction.dto';
+
+@Controller('auctions')
+export class AuctionsController {
+  constructor(private readonly auctionsService: AuctionsService) {}
+
+  @Post()
+  create(@Body() createAuctionDto: CreateAuctionDto) {
+    return this.auctionsService.create(createAuctionDto);
+  }
+
+  @Get()
+  findAll() {
+    return this.auctionsService.findAll();
+  }
+
+  @Get(':id')
+  findOne(@Param('id') id: string) {
+    return this.auctionsService.findOne(+id);
```

Comment:
Do we use integer values? Or numeric? This +ID converts to number.

If we use numeric, you can add pipeline to verify it's proper number (positive and numeric).

Reply:
I wanted to add pipelines in the next PR (or PRs) 🤓